packages/tracing-internal/src/browser/request.ts

```diff
@@ -214,7 +214,7 @@ export function shouldAttachHeaders(url: string, tracePropagationTargets: (strin
  *
  * @returns Span if a span was created, otherwise void.
  */
-function fetchCallback(
+export function fetchCallback(
```

Comment:
l: why is this change necessary?

Reply:
It was because this was actually being imported by tests: https://github.com/getsentry/sentry-javascript/blob/2cf8a6a8e6879249f29ff77e82b67b3bdad4ce5d/packages/tracing-internal/test/browser/request.test.ts#L10